version.json

```diff
@@ -4,14 +4,14 @@
     "annotated git tag without the leading 'v'.",
     "This value gets updated automatically by `scripts/prepare-release`."
   ],
-  "tenzir-version-fallback": "4.0.0-rc3",
+  "tenzir-version-fallback": "4.0.0-rc4",
   "tenzir-version-rev-count_COMMENT": [
     "This value stores the ancestor count of the tagged commit, calculated",
     "with `git rev-list --count HEAD`, then incremented by 1. This operates",
     "under the assumption that the release-preparing PR contains exactly one",
     "commit and is rebased before merging."
   ],
-  "tenzir-version-rev-count": 17208,
+  "tenzir-version-rev-count": 17247,
```

Comment:
There are two approved PRs on auto-merge, we must synchronize updating this number with them being merged.